src/d3a/models/market/two_sided.py

```diff
@@ -252,10 +252,10 @@ def match_recommendation(self, recommended_list):
                 propagated_bid_rate=bid.price/bid.energy,
                 original_offer_rate=offer.original_offer_price/offer.energy,
                 propagated_offer_rate=offer.price/offer.energy,
-                trade_rate=recommended_pair.trade_rate)
+                trade_rate=original_bid_rate)
```

Comment:
Can you please explain this, change too? Why is always the `original_bid_rate` used for the trade_rate? I would expect the `bid.energy_rate`  for the pay-as-bid and the `clearing_rate` for the pay-as-clear market. Consequently, I would expect that the original implementation is correct. Right?

Reply:
OK, thanks a lot! Indeed, I wasn't fully aware of the concept of `TradeBidOfferInfo is a structure that is referring to the end-to-end trade chain [..] and not to the trade of this market` (any more, probably self-protection of my brain ;-) ).
Nevertheless. I took some time and looked into the implementation again. I also remember the discussion about the nomenclature now and that I was not a fan of naming it just 'trade_rate'. Anyway. Got it now (i think ;-)) .Thanks!